extensions/roulette/roll/cog.py

```diff
@@ -162,6 +172,20 @@ async def _determine_targets(self, message: Message) -> Set[Member]:
         self.logger.debug("Didn't find any mentions, returning message author as target.")
         return {message.author}
 
+    # TODO: Check for successful update
+    async def _upload_timeout(self,
+                              duration: timedelta,
+                              message: Message,
+                              target: Member) -> int:
+
+        key = "roulette_timeout_live_" + str(message.guild.id)
```

Comment:
note: This key should be a file-level constant (which will be defined in the roulette redis file following the comment above).

(Keep this `_upload_timeout` method; just move the `zadd` method into the separate API)